src/Consolonia.Core/Infrastructure/ConsoleWindow.cs

```diff
@@ -469,5 +482,118 @@ protected virtual void Dispose(bool disposing)
                 _disposedValue = true;
             }
         }
+
+        /// <summary>
+        ///     This works by creating a Software cursor, aka a sprite that is drawn on top of the screen.
+        ///     It draws the current cursor directly to the console buffer, but maintains a list of the pixels need to be redrawn
+        ///     whenever the cursor moves.
+        /// </summary>
+        /// <param name="point"></param>
+        private void RenderSoftwareCursor(Point point)
```

Comment:
Would not be simplier to draw a symbol in the overlay?

Reply:
I tried that first, all sorts of issues with clipping, getting in the way of eventing, etc